### PR TITLE
Update the EDK2 toolchain and QEMU invoking commands

### PR DESCRIPTION
**Pull request by xiangmy**
As discussed in https://github.com/OP-TEE/manifest/issues/179
+ Older EDK2 toolchains (GCC49) will cause errors when updating EDK2, therefore, we use GCC5 instead.
+ QEMU has adopted meson as the build system. As a result, the output of QEMU will be located in the $(QEMU_PATH)/build, so we should reflect this change.

### CODE REVIEW

File: qemu.mk [hunk 1]
```diff
@@ -195,7 +195,7 @@ endif
 check: $(CHECK_DEPS)
 	ln -sf $(ROOT)/out-br/images/rootfs.cpio.gz $(BINARIES_PATH)/
 	cd $(BINARIES_PATH) && \
-		export QEMU=$(ROOT)/qemu/arm-softmmu/qemu-system-arm && \
+		export QEMU=$(ROOT)/build/qemu/arm-softmmu/qemu-system-arm && \
```
Comment (jforissier) [inline qemu.mk:198]: `$(QEMU_PATH)` not `$(ROOT)`

**Review by jforissier — CHANGES_REQUESTED** [review]
Thanks for the patch. Please create one commit for the EDK2 update and one for QEMU and see below.

File: qemu.mk (hunk 1 above)
Comment (xiangmy) [reply qemu.mk]: Fixed, thanks!

Comment (jforissier) [reply qemu.mk]: And you forgot to update `$(QEMU_PATH)/config-host.mak:`.
I suggest you define `QEMU_BUILD ?= $(QEMU_PATH)/build` just after `QEMU_PATH` then use that where needed.

**Comment by jforissier** [comment]
For commit "Update the EDK2 toolchain":
`Reviewed-by: Jerome Forissier <jerome@forissier.org>`
`Tested-by: Jerome Forissier <jerome@forissier.org> (QEMUv8)`

File: qemu.mk (hunk 1 above)
Comment (xiangmy) [reply qemu.mk]: Thanks for the advice! I've made the changes.

**Comment by jforissier** [comment]
For commit "Update QEMU invoking commands":
Please add prefix: "qemu, qemu_v8: update QEMU invoking commands". With that:
`Reviewed-by: Jerome Forissier <jerome@forissier.org>`

Please also add my review/test tags.

**Comment by xiangmy** [comment]
Made the changes, thanks!

**Review by jforissier — APPROVED** [review]

**Pull request merged**